src/array-practice/task3.js

```diff
@@ -10,4 +10,7 @@
  * forEachRight([1, 2, 3, 4], val => console.log(val)); -> в консоль 4 3 2 1
  */
 
-export const forEachRight = (array, fn) => array.reverse().forEach(fn);
\ No newline at end of file
+export const forEachRight = (array, fn) => {
+  const buffer = [...array];
+  buffer.reverse().forEach(fn);
```

Comment:
everything can be written in 1 line. usually we puttin something in variable if
- logic is complex
- variable is reused